_layouts/tutorial_hands_on.html

```diff
@@ -59,10 +59,11 @@
                 <ul class="navbar-nav">
                     <li class="nav-item">
                         <a class="nav-link" href="{{ site.baseurl }}/topics/{{ topic.name }}" title="Go back to list of tutorials">
-                            {% icon topic %} {{ topic.title }}
+                            {% icon topic %} Back to Topic
```

Comment:
I prefer the topic title. The "Back to" seems to assume one reached the tutorial from the topic page (also the `title` on the line above), but I image often people would get to the tutorial page from a search engine or a direct link from a course program.

Reply:
how about "Topic page" then orso? I mainly did it because for some of the long topic names it takes up a lot of screen real estate

Reply:
and a little bit bacause I if it just says "Metagenomics" at the top, it wasn't 100% obvious this meant it would take you to the topic tutorial list orso?

Reply:
To save screen real state, maybe this can be removed from the top menu and we can add breadcrumb links immediately below, e.g.

[Galaxy for Scientists]() > [Introduction to Galaxy Analyses]()

Otherwise the icon here could be changed to https://fontawesome.com/icons/level-up , which I think it's already used in the slide decks.

Just ideas for a future PR.

Reply:
Oh, nice idea. I like that. Easy to include above the tutorial title and makes more sense there.